src/EncompassRest/CustomDataObjects/CustomDataObject.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace EncompassRest.CustomDataObjects
+{
+    public sealed class CustomDataObject : ExtensibleObject
+    {
+        public string Name { get; set; }
+
+        public byte[] DataObject { get; set; }
```

Comment:
should there be a DataObject.ToString() for simplicity?  or just leave that to the client?

Reply:
or should it be a string?  more often than not, i am not storing binary data, just a string in a file.  let the users that need it as a byte[] convert?

Reply:
I think we should leave that to the client, because then we'd be assuming the file types they're using as well as assuming a certain Encoding.